types/tedious/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for tedious 3.0.0
+// Type definitions for tedious 4.0.0
```

Comment:
These are not breaking changes so there's no need to increment the major version, everything else is great.

Reply:
I did intend to remove the deprecated properties so did this in anticipation, then decided I wouldn't do that and forgot to decrement the version number.

I will submit another PR to remove the deprecated properties.

Reply:
Actually, looks like that's already been done